src/Dflydev/FigCookies/SetCookie.php

```diff
@@ -165,6 +165,29 @@ public static function create($name, $value = null)
         return new static($name, $value);
     }
 
+    /**
+     * Create a cookie that lives "forever" (for five years).
+     *
+     * @param string $name
+     * @param string $value
+     * @return SetCookie
+     */
+    public static function forever($name, $value = null)
+    {
+        return static::create($name, $value)->withExpires(new DateTime('+5 years'));
```

Comment:
Is +5 years standard for "forever?" If not, this makes me feel a little uncomfortable but I like the direction!